assets/terraform-modules/matchbox-flatcar/variables.tf

```diff
@@ -0,0 +1,72 @@
+variable "http_endpoint" {
+  type        = string
+  description = "Matchbox HTTP read-only endpoint (e.g. http://matchbox.example.com:8080)."
+}
+
+variable "os_channel" {
+  type        = string
+  description = "Flatcar Container Linux channel to install from (stable, beta, alpha, edge)."
+  default     = "stable"
+}
+
+variable "os_version" {
+  type        = string
+  description = "Flatcar Container Linux version to install (for example '2191.5.0' - see https://www.flatcar-linux.org/releases/)."
+  default     = "current"
+}
+
+variable "download_protocol" {
+  type        = string
+  description = "Protocol iPXE should use to download the kernel and initrd. Defaults to https, which requires iPXE compiled with crypto support. Unused if cached_install is true."
+  default     = "https"
+}
+
+variable "cached_install" {
+  type        = bool
+  description = "Whether the operating system should PXE boot and install from matchbox /assets cache. Note that the admin must have downloaded the os_version into matchbox assets."
+  default     = false
+}
+
+variable "install_disk" {
+  type        = string
+  description = "Disk device to which the install profiles should install the operating system (e.g. /dev/sda)."
+  default     = "/dev/sda"
+}
+
+variable "container_linux_oem" {
+  type        = string
+  description = "DEPRECATED: Specify an OEM image id to use as base for the installation (e.g. ami, vmware_raw, xen) or leave blank for the default image."
+  default     = ""
+}
+
+variable "kernel_args" {
+  description = "Additional kernel arguments to provide at PXE boot."
+  type        = list(string)
+  default     = []
+}
+
+variable "install_to_smallest_disk" {
+  description = "Install Flatcar Container Linux to the smallest disk."
+  type        = bool
+  default     = false
+}
```

Comment:
Type and description fields are in the wrong order here still.

Reply:
oops, my bad. I wasn't thorough.